tests/python/contrib/test_hexagon/test_benchmark_elemwise_add.py

```diff
@@ -20,25 +20,22 @@
 import os.path
 import sys
 import tempfile
-
```

Comment:
Nit: Can we keep the newline separation between python standard libraries and 3rd-party module imports?

Reply:
done